texastribune/donations-app
# Add FAQ page

#### What's this PR do?

Adds the FAQ page into this app instead of linking out to /support-us/frequently-asked-questions/ on the Tribune site.
#### Why are we doing this? How does it help us?

So that people wanting to learn more about donating will stay inside of our 'Support Us' app experience instead of being linked out to FAQ on the Tribune site. Also, the FAQ mechanism on the current site is not intuitive (it was a headache to update recently) and it will be nice to not have that be the method for updating FAQ.
#### How should this be manually tested?

Visit the 'Support Us' page manually. Click on 'Frequently Asked Questions' in the nav, and you should remain in the app and see the Frequently Asked Questions page come up (/faq.html). You should see the updated questions there that match what's currently on the site. It should be responsive at all sizes, so play around with that and check that out. Confirm that the links in the questions are all what you would expect.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

Not really.
#### What are the relevant tickets?

Part of Sprint task to clean out /about/donors code: https://basecamp.com/1866004/projects/1175549/todos/176840690
#### Next steps?

To remove FAQ from the Tribune project.

To add updating with Google docs so that it doesn't require any code pushes for FAQ changes.
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?
#### Technical debt note

Same here / less on Tribune
